test/sanitytest.c

```diff
@@ -49,6 +49,15 @@ static int test_sanity_enum_size(void)
     return 1;
 }
 
+static int test_sanity_int_size(void)
+{
+    if (!TEST_size_t_eq(sizeof(int), 4)
+        || !TEST_int_eq(CHAR_BIT, 8)
+        || !TEST_size_t_lt(sizeof(int), sizeof(int64_t)))
```

Comment:
I would be surprised if 64-bit 'int' happened, yes.  A lot of things would break (0x7fffffff constants, etc.).
Though we can go with just the one le(int, int64) check if you still want to go that way.